test/unit/test_unit.ml

```diff
@@ -122,4 +122,4 @@ let tests =
   ; ("Range", Test_range.tests)
   ; ("Translation_unit", Test_translation_unit.tests) ]
 
-let () = Alcotest.run "ocamlformat" tests
+let () = Alcotest.run ~argv:[|""; "--compact"|] "ocamlformat" tests
```

Comment:
I don't know the consequences of overriding the argv here but there's a `~compact:true` argument that seems better.

Reply:
Oh I missed this arg, good catch!